src/Sarif.Multitool/PageOptions.cs

```diff
@@ -50,7 +50,7 @@ internal class PageOptions
             "force",
             Default = true,
             HelpText = "Force overwrite of output file if it exists.")]
-        public bool Force { get; set; } = true;
```

Comment:
This fixes #1630, "Multitool page command --force defaults to true and can't be changed."